R/broom_wrapper.R

```diff
@@ -88,7 +88,7 @@ add_prediction <- function(df, model_df, conf_int = 0.95, ...){
   aug_args <- expand_args(cll, exclude = c("df", "model_df"))
 
   # validate data frame based on meta info
-  model_meta <- model_df[[".model_meta_information"]]
+  model_meta <- model_df[[".model_meta_data"]]
```

Comment:
Sorry for being picky on the name, but I think it's usually "metadata" than "meta data".